.travis.yml

```diff
@@ -65,6 +65,10 @@ matrix:
               - install_test cant-find "Http\Discovery\HttpClientDiscovery::find();" "symfony/http-client:5.* php-http/httplug guzzlehttp/psr7:1.* http-interop/http-factory-guzzle"
               # We should be able to find a client when Symfony is only partly installed and we have guzzle adapter installed
               - install_test will-find "Http\Discovery\HttpClientDiscovery::find();" "symfony/http-client:5.* php-http/guzzle6-adapter php-http/httplug php-http/message-factory guzzlehttp/psr7:1.*"
+              # Test that we find a client with Symfony and Guzzle
+              - install_test will-find "Http\Discovery\HttpClientDiscovery::find();" "friendsofsymfony/http-cache-bundle:2.4.* symfony/http-client:4.* php-http/guzzle6-adapter"
+              # Test that we find an async client with Symfony and Guzzle
+              - install_test will-find "Http\Discovery\HttpAsyncClientDiscovery::find();" "friendsofsymfony/http-cache-bundle:2.4.* symfony/http-client:4.* php-http/guzzle6-adapter"
```

Comment:
You can replace `friendsofsymfony/http-cache-bundle:2.4.*` with `php-http/client-common:2.* php-http/message:1.8*`

It is the same effect.